README.md

```diff
@@ -36,7 +36,7 @@ Tools built on DynamoRIO and available in the [release package](https://dynamori
   - [Function call tracing](https://dynamorio.org/sec_drcachesim_tools.html#sec_tool_func_view)
 - The legacy processor emulator
   [drcpusim](https://dynamorio.org/page_drcpusim.html)
-- The "strace for Windows" tool [drstrace](http://drmemory.org/strace_for_windows.html)
+- The "strace for Windows" tool [drstrace](https://dynamorio.org/page_drstrace.html)
```

Comment:
Plus, the "this link" at https://dynamorio.org/page_drstrace.html which is trying to work for a local file view (a downloaded unpacked zip package), fails online: best to also provide https://drmemory.org/page_drstrace.html there in addition to the local-file link so at least one of them will work.

Reply:
I'll do so. There are a few others places in the documentation where we have it.

Reply:
OK, thank you for the contribution.